packages/notifications/src/Notification.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+namespace Filament\Notifications;
+
+use Filament\Support\Components\ViewComponent;
+use Livewire\Wireable;
+use Illuminate\Support\Str;
+
+class Notification extends ViewComponent implements Wireable
+{
+    protected string $view = 'notifications::components.notification';
+
+    public string $id;
+
+    public string $title;
+
+    public ?string $description = null;
+
+    public ?string $icon = null;
+
+    public ?string $status = null;
+
+    public function __construct()
+    {
+        $this->id(Str::uuid());
+    }
+
+    public static function make(): static
+    {
+        $static = app(static::class);
+        $static->setUp();
+        $static->success();
```

Comment:
This configuration should be in the `setUp()` method

Reply:
Whoops, f01b3138fcbc94dc3a12d95727ef5dd955305dcb.